Fix version. Placeholder will be replaced by post-patch in Portfile.
--- configure.ac.orig	2021-08-13 23:18:24.000000000 -0500
+++ configure.ac	2023-03-22 00:15:21.000000000 -0500
@@ -1,6 +1,6 @@
 dnl Process this file with autoconf to produce a configure script.
 
-AC_INIT(GDash, 20180129unstable, [https://bitbucket.org/czirkoszoltan/gdash])
+AC_INIT(GDash, @VERSION@, [https://bitbucket.org/czirkoszoltan/gdash])
 AC_CONFIG_SRCDIR([src/main.cpp])
 AM_INIT_AUTOMAKE
 AM_CONFIG_HEADER(config.h)
